conf/base/bblayers.conf

```diff
@@ -18,7 +18,7 @@ BBLAYERS += "${BSPDIR}/sources/meta-uav"
 BBLAYERS += "${BSPDIR}/sources/meta-ros"
 BBLAYERS += "${BSPDIR}/sources/meta-intel-aero"
 BBLAYERS += "${BSPDIR}/sources/meta-intel-aero-connectivity"
-BBLAYERS += "${BSPDIR}/sources/meta-intel-realsense"
+BBLAYERS += "${BSPDIR}/sources/meta-intel-aero-librealsense"
```

Comment:
it was not really a typo. The layer changed. I pushed everything on Friday to get it working and forgot to push the change to the manifest, so this is probably why it failed for you. See https://github.com/intel-aero/meta-intel-aero/issues/61